src/hooks/context/hooks.tsx

```diff
@@ -0,0 +1,103 @@
+import { Context, ContextFileNotFoundError, ContextNotFoundError, MissingCurrentContextError } from './models';
+import { ContextService } from './contextService';
+import { container } from 'tsyringe';
+import { SpecificationFile } from '../validation';
+
+export const useContextFile = () => {
+	const contextService: ContextService = container.resolve(ContextService);
+
+
+	return {
+		list: () => {
+			try {
+				let ctx: Context = contextService.loadContextFile();
+				let response = Object.keys(ctx.store).map(c => ({ key: c, path: ctx.store[c] }));
+				return { response };
+			} catch (error) {
+				return { undefined, error };
```

Comment:
I will type this kind of returned object in order to give some behaviour like the ValidationResponses